packages/react/src/app-layout/AppLayoutHeaderBrand.tsx

```diff
@@ -67,3 +76,30 @@ export function AppLayoutHeaderBrand({
 		</Flex>
 	);
 }
+
+type AppLayoutHeaderBrandBadgeProps = PropsWithChildren<{}>;
+
+// Note: This component has been copied over from the `Header` component
+function AppLayoutHeaderBrandBadge({
```

Comment:
We *could* export/import `HeaderBadge` from the `Header` component here to avoid some duplication... but I've opted to copy/paste in case we need to tweak font sizes etc independently.